StoreApp/Views/Store/DetailsProduct.cshtml

```diff
@@ -92,7 +92,9 @@
 
     </dl>
 </div>
+
```

Comment:
My proposition to remove empty space between tags.

Reply:
fixed